py/picca/utils.py

```diff
@@ -77,6 +78,88 @@ def smooth_cov(da,we,rp,rt,drt=4,drp=4):
     co_smooth = cor_smooth * sp.sqrt(var*var[:,None])
     return co_smooth
 
+def smooth_cov_wick(da,we,cow,np,nt):
```

Comment:
@vserret, I think the function should have as input the path to the covariance matrix and the path to where to save the new covariance. Then the function reads the wick covariance, does it thing and then writes it to a fits file, that can be used in export.py. What do you think?

Reply:
ok I will do that